cmd/extract/main.go

```diff
@@ -206,8 +212,20 @@ func main() {
 	}
 }
 
-func mnemonicToSeed(mnemonic string) []byte {
-	return pbkdf2.Key([]byte(mnemonic), []byte("mnemonic"), 2048, 64, sha512.New)
+func mnemonicToSeed(mnemonic string) ([]byte, error) {
+	phrases := strings.Split(mnemonic, " ")
+
+	if len(phrases) != 12 {
+		return nil, fmt.Errorf("12 mnemonics needed, yet %d given", len(phrases))
+	}
+
+	for _, phrase := range phrases {
+		if _, ok := internal.Bip39Words[phrase]; !ok {
+			return nil, fmt.Errorf("invalid mnemonic given (case-sensitive): '%s'", phrase)
```

Comment:
I think it might be clearer if we don't wrap the phrase in quotes:
```suggestion
			return nil, fmt.Errorf("invalid mnemonic given (case-sensitive): %s", phrase)
```